src/app/dashboard/dashboard.component.html

```diff
@@ -1,8 +1,9 @@
-<div fxLayout="column" fxLayout.gt-md="row" fxLayoutGap="10px">
-  <mat-card fxFlex>
-    <app-graph [chartType]="'line'"></app-graph>
-  </mat-card>
-  <mat-card fxFlex>
-    <app-graph [chartType]="'bar'"></app-graph>
+<div class="flex-container" fxflex>
+  <mat-card class="flexitem" *ngFor="let product of products ">
+      <figure>
+        <img  [src]="product.imageUrl" alt="{{product.description}}" style='width:110%;'>
```

Comment:
110% width seems suspect. What are you trying to accomplish here? Also we should move inline styles to a scss file.